### PR TITLE
[cinder]Support to remove dependence of volume on its snapshot

### PR DESCRIPTION
**Pull request by sjamgade**

### CODE REVIEW

File: chef/cookbooks/cinder/templates/default/cinder.conf.erb [hunk 1]
```diff
@@ -207,6 +207,9 @@ volume_driver = cinder.volume.drivers.lvm.LVMVolumeDriver
   <% end -%>
 
   <% if volume['backend_driver'] == 'rbd' -%>
+    <% if volume['rbd']['flatten_volume_from_snapshot']  -%>
```
Comment (aplanas) [inline chef/cookbooks/cinder/templates/default/cinder.conf.erb]: extra space here at the end

File: chef/data_bags/crowbar/template-cinder.schema [hunk 2]
```diff
@@ -246,7 +246,8 @@
                       "admin_keyring": { "type": "str", "required": true },
                       "pool": { "type": "str", "required": true },
                       "user": { "type": "str", "required": true },
-                      "secret_uuid" : { "type": "str", "required": true }
+                      "secret_uuid" : { "type": "str", "required": true },
+                      "flatten_volume_from_snapshot": { "type": "bool", "required": false }
```
Comment (aplanas) [inline chef/data_bags/crowbar/template-cinder.schema]: why is not required? I wonder if is not better to do a required one, add the migration and set as a default value 'false' here

**Comment by dirkmueller** [comment]
@sjamgade I think the style of the commit message summary should be ```cinder: ...``` not ```[cinder] ...```

File: chef/cookbooks/cinder/templates/default/cinder.conf.erb (hunk 1 above)
Comment (sjamgade) [inline chef/cookbooks/cinder/templates/default/cinder.conf.erb]: done https://github.com/crowbar/crowbar-openstack/pull/883

File: chef/data_bags/crowbar/template-cinder.schema (hunk 2 above)
Comment (sjamgade) [inline chef/data_bags/crowbar/template-cinder.schema]: done https://github.com/crowbar/crowbar-openstack/pull/883

**Comment by sjamgade** [comment]
recreated https://github.com/crowbar/crowbar-openstack/pull/883 from a branch

**Pull request closed without merging**